### PR TITLE
add new function to request component without loading the container again

### PR DESCRIPTION
**Pull request by chensixx**
task3118
## Description

Using resolvedUrl on the container and the same logic in loader.request, to create a new function that can load component without loading the container again

### CODE REVIEW

File: packages/loader/container-loader/src/loader.ts [hunk 1]
```diff
@@ -350,6 +350,29 @@ export class Loader implements IHostLoader {
 		);
 	}
 
+	/**
+	 * With an already-resolved container, we can request a component directly, without loading the container again
+	 * @param container - a resolved container
+	 * @returns component on the container
+	 */
+	public async requestResolvedObjectFromContainer(container: IContainer): Promise<IResponse> {
```
[the remark this answers is not in the document]
Comment (anthony-murphy) [reply packages/loader/container-loader/src/loader.ts:358]: that will also keep this from being a breaking change

[the remark this answers is not in the document]
Comment (anthony-murphy) [reply packages/loader/container-loader/src/loader.ts:358]: @alexvy86 did you deprecate request on the loader? this new function would be the replacement. At some point we could integrate checking the entrypoint for a router here as well, and giving priority to that.

[the remark this answers is not in the document]
Comment (alexvy86) [reply packages/loader/container-loader/src/loader.ts:358]: No, we didn't deprecate the request methods at the container layer yet. With this in place we could now do it and point here.

File: packages/loader/container-loader/src/loader.ts [hunk 2]
```diff
@@ -257,6 +257,26 @@ export type IDetachedBlobStorage = Pick<IDocumentStorageService, "createBlob" |
...
 };
 
+/**
+ * With an already-resolved container, we can request a component directly, without loading the container again
+ * @param container - a resolved container
+ * @returns component on the container
+ */
+export async function requestResolvedObjectFromContainer(
+	container: IContainer,
+): Promise<IResponse> {
```
Comment (anthony-murphy) [inline packages/loader/container-loader/src/loader.ts:267]: i think adding an optional headers param could be useful, if they need to pass any.  `headers?: IRequestHeader`

Comment (anthony-murphy) [reply packages/loader/container-loader/src/loader.ts:267]: some examples of the types of headers someone may want to pass: https://github.com/microsoft/FluidFramework/blob/main/packages/runtime/container-runtime/src/containerRuntime.ts#L1570L1578

**Review by anthony-murphy — APPROVED** [review]

**Pull request merged**